Packs/Hoxhunt/Integrations/Hoxhunt/Hoxhunt.py

```diff
@@ -0,0 +1,365 @@
+        }
+        return self.query(mutation, variables)
+
+    def set_incident_sensitive(self, incident_id: str, is_sensitive: bool) -> Dict[str, Any]:
+        mutation = '''
+        mutation SetIncidentSensitive($incidentId: String!, $isSensitive: Boolean!) {
+            setIncidentSensitive(
+                incidentId: $incidentId
+                isSensitive: $isSensitive
+            ) {
+                _id
+                hasSensitiveInformation
+            }
+        }
+        '''
+        variables = {
+            'incidentId': incident_id,
+            'isSensitive': is_sensitive
+        }
+        response = self.query(mutation, variables)
+        if 'errors' in response:
+            raise ValueError(f"Failed to set incident sensitive: {response['errors']}")
+        data = response.get('data', {}).get('setIncidentSensitive')
+        if data is None:
+            raise ValueError("Failed to set incident sensitive: No data returned.")
+        return {
+            '_id': data.get('_id'),
+            'hasSensitiveInformation': data.get('hasSensitiveInformation'),
+        }
+
+    def set_incident_soc_classification(self, incident_id: str, classification: str) -> Dict[str, Any]:
+        mutation = '''
+        mutation SetIncidentSocClassification($incidentId: String!, $classification: SocClassification!) {
+            setIncidentSocClassification(
+                incidentId: $incidentId
+                classification: $classification
+            ) {
+                _id
+                humanReadableId
+                updatedAt
+                socClassification
+            }
+        }
+        '''
+        variables = {
+            'incidentId': incident_id,
+            'classification': classification
+        }
+        return self.query(mutation, variables)
+
+    def update_incident_state(self, incident_id: str, state: str) -> Dict[str, Any]:
+        mutation = '''
+        mutation UpdateIncidentState($incidentId: ID!, $state: IncidentState!) {
+            updateIncidentState(
+                incidentId: $incidentId
+                state: $state
+            ) {
+                _id
+                humanReadableId
+                updatedAt
+                state
+            }
+        }
+        '''
+        variables = {
+            'incidentId': incident_id,
+            'state': state
+        }
+        return self.query(mutation, variables)
+
+def test_module(client, query) -> str:
+    result = client.query(query)
+    if result:
+        return 'ok'
+    else:
+        return 'Test failed: ' + str(result)
+
+
+
+def create_output(results: Dict[str, str], endpoint: str, keyfield: str = '') -> CommandResults:
+    human_readable = tableToMarkdown('Hoxhunt results', results)
+    output = CommandResults(
+        outputs_prefix=f'Hoxhunt.{endpoint}',
+        outputs_key_field=keyfield,
+        outputs=results,
+        readable_output=human_readable
+    )
+    return output
```

Comment:
remove this line
(no need to create this output variable, just return the CommandResults